src/workflows/dags/forecast_prediction_jobs.py

```
@@ -18,16 +18,16 @@
     tags=["airqo", "hourly-forecast", "daily-forecast", "prediction-job"],
 )
 def make_forecasts():
-    bucket = configuration.FORECAST_MODELS_BUCKET
-    project_id = configuration.GOOGLE_CLOUD_PROJECT_ID
+    bucket = Config.FORECAST_MODELS_BUCKET
+    project_id = Config.GOOGLE_CLOUD_PROJECT_ID
 
     ### Hourly forecast tasks
     @task()
     def get_historical_data_for_hourly_forecasts():
         from datetime import datetime, timedelta, timezone
```

Ruff (0.8.2)

Redefinition of unused `datetime` from line 3

Remove definition: `datetime`

(F811)